### DISCUSSION TITLE
the position of the mouse

### DISCUSSION THREAD

**Discussion by branaway** [pixiv/three-vrm#1107] [Help]
Can I ask a simple question? How can I find out the world position of the mouth of the VRM model in a scene?

**Comment by branaway** [comment]
sorry for my slippery finger. I should have been mouth in the title.

**Comment by 0b5vr** [comment]
Unfortunately, you cannot get the exact position of the mouth of avatars.

However, you can get a position of the head bone (located at the root of the head) instead.
Try having a hardcoded offset (say, `(0.0, 0.03, 0.03)`) and use [`Object3D.localToWorld`](https://threejs.org/docs/#api/en/core/Object3D.localToWorld), like this code below (not tested):

```js
const headNode = vrm.humanoid.getNormalizedBoneNode('head');
const offset = new THREE.Vector3(0.0, 0.03, 0.03);
headNode.localToWorld(offset);
```
